doc/API.md

```diff
@@ -27,6 +27,32 @@ Example GET curl call:  `curl -A "Sia-Agent" /wallet/transactions?startheight=1&
 
 Example POST curl call: `curl -A "Sia-Agent" --data "amount=123&destination=abcd" /wallet/siacoins
 
+Standard responses
+------------------
+
+#### Success
+
+The standard response indicating the request was successfully processed is HTTP
+status code 204.
+
+#### Error
+
+The standard error response indicating the request failed for any reason, is a
+4xx HTTP status code with the error message as plain text.
```

Comment:
I believe that the error message should be returned in json, that way additional objects can be added for compat support or other weirdness.

This isn't the link I was thinking of, but it does talk about error design:  https://apigee.com/about/blog/technology/restful-api-design-what-about-errors